samples/HelloAndroid/codebase/HelloAndroid.sharpmake.cs

```diff
@@ -12,24 +12,31 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using System.IO;
 using Sharpmake;
 
 namespace HelloAndroid
 {
     [Sharpmake.Generate]
     public class HelloAndroidSolution : CommonSolution
     {
+        public string GradleRootPath = Path.Combine(Path.Combine(Globals.TmpDirectory, @"..\.."), "gradle/root");
```

Comment:
Too many path.combine and mix and match of slashing, change to
`Path.Combine(Globals.TmpDirectory, @"..\..\gradle\root");`